iam.tf

```diff
@@ -25,6 +26,10 @@ locals {
         "batcave/sonar-registry-credentials",
         "batcave/sonar-agent-api-key",
         "batcave/kiali",
-        "batcave/gitlab-rails-secret-s3",
+        "batcave/gitlab-rails-secret-s3"
```

Comment:
Why did the trailing comma get stripped?

Reply:
Added a few arns, there doesn't need to be a comma after the last entry in the list right?